AUTHORS

```diff
@@ -40,5 +40,4 @@ Tom Evans
 Dylan Giesler
 Spencer Carroll
 Dulmandakh Sukhbaatar
-Rustem Saiargaliev
```

Comment:
This should have been kept :)

Reply:
That's alright, I hope I'll get another chance to add something here! 👍